lib/MySQL_Session.cpp

```diff
@@ -1514,9 +1513,10 @@ int MySQL_Session::handler_again___status_RESETTING_CONNECTION() {
 						myconn->parent->port,
 						myds->fd,
 						myds->myconn->fd,
-						myerr,
+						ER_PROXYSQL_OFFLINE_SRV,
```

Comment:
This can stay `myerr` .
For the same replace didn't happen in othe functions, like `handler_again___status_SETTING_INIT_CONNECT`

Reply:
If this stays `myerr` error code will just be `0`, for being consistent with an error code based approach, maybe we should also change it in `handler_again___status_SETTING_INIT_CONNECT` (I missed that one during the changes).